apix-rest-v1/src/main/java/eu/xenit/apix/rest/v2/people/PeopleWebscript.java

```diff
@@ -43,6 +43,11 @@ public void getPerson(@UriVariable final String space, @UriVariable final String
             @UriVariable final String guid, WebScriptResponse webScriptResponse) throws IOException {
         logger.debug("Asked person with guid: " + guid);
         Person p = personService.GetPerson(createNodeRef(space, store, guid));
+        if (p == null) {
+            webScriptResponse.setStatus(404);
+            webScriptResponse.getWriter().write("Person does not exist");
```

Comment:
Hmm, returning a non-JSON body as an error response in a JSON API?

Either returning no body (404 already implies that the requested resource doesn't exist), or a JSON object would be fine, I think

Reply:
Ok. I just copied the code over. Now added a contentType.